SPECS/bash/bash.spec

```diff
@@ -328,6 +330,10 @@ fi
 %defattr(-,root,root)
 
 %changelog
+* Mon Jun 17 2024 Daniel McIlvaney <damcilva@microsoft.com> - 5.2.15-2
+- When non-interactive shells are started with '-bash' load startup files. From
+  Fedora upstream: https://src.fedoraproject.org/rpms/bash/blob/f40/f/bash-2.03-profile.patch
```

Comment:
can you add a dash to the start of this comment?

Reply:
This is auto-formatted via `rpmdev-bumpspec`, but I can add.